packages/cli/src/collections/command.ts

```diff
@@ -237,6 +244,61 @@ const items = {
   },
 };
 
+// In your options definitions (collections/command.ts)
+const googleSheetOptions = [
+  {
+    name: 'sheet',
+    yargs: {
+      description: 'Google Sheet URL or ID',
+    },
+  },
+  {
+    name: 'range',
+    yargs: {
+      description:
+        'Range of cells to read from the Google Sheet -  default: A0:B0. First column is the key.',
+      default: 'A1:Z',
+    },
+  },
+  {
+    name: 'skip-headers',
+    yargs: {
+      description: 'Skip the first row of the Google Sheet (headers)',
+      type: 'boolean',
+      default: false,
+    },
+  },
+  {
+    name: 'worksheet-name',
+    yargs: {
+      description: 'Name of the worksheet to use (default: "Sheet1")',
+      default: 'Sheet1',
+    },
+  },
+  {
+    name: 'key-column',
+    yargs: {
+      description: 'Column to use as key (either index or header name)',
+      default: 0,
+    },
+  },
+  {
+    name: 'value-column',
+    yargs: {
+      description: 'Column to use as value (either index or header name)',
+      default: 1,
+    },
+  },
+  // We can have this later if we want
```

Comment:
Yep - I fear this will be quite hard. I'll ask around and see if anyone has any insight.